tests_cypress/config.js

```diff
@@ -0,0 +1,52 @@
+const config = {
+    STAGING: {
+        API: {
+            HostName: "https://api.staging.notification.cdssandbox.xyz",
+        },
+        Admin: {
+            HostName: "https://qvgi7ansvfzeekqiuv6hltlzza0sotpj.lambda-url.ca-central-1.on.aws/", //"https://staging.notification.cdssandbox.xyz",
+            AdminUser: "eli10@ethereal.email",
```

Comment:
Brings a good insight: we should put domain validation on login (on top of existing registration validation).